modules/core/src/main/scala/io/circe/jackson/snowplow/package.scala

```diff
@@ -34,7 +34,7 @@ package object snowplow {
    * @param json instance of circe's Json
    * @return converted JsonNode
    */
-  final def circeToJackson(json: Json): JsonNode =
+  final def circeToJackson(json: Json, maxDepth: Int): JsonNode =
```

Comment:
**TODO**: Currently `maxDepth` is decremented on each recursion, but it is otherwise ignroed.  Change this function so it returns an error if `maxDepth` decreased below zero.